processor/k8sattributesprocessor/README.md

```diff
@@ -271,10 +271,16 @@ k8sattributes/2:
       - k8s.node.name
       - k8s.pod.start_time
     labels:
-     # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
-     - tag_name: app.label.component
-       key: app.kubernetes.io/component
-       from: pod
+      # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
+      - tag_name: app.label.component
+        key: app.kubernetes.io/component
+        from: pod
+    automatic_attributes:
+      # Apply the operator rules - see https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md
+      enabled: true
+      well_known_labels: true # default is false
+      annotation_prefixes: ["foo/"] # default is ["resource.opentelemetry.io/"] - use empty list to disable
+      exclude: ["service.version"] # default is empty list
```

Comment:
Can we just move well known attributes from this section to where all the other pre-defined resource attributes are configured `extract.metadata`, and disabled by default?

Then this section would be responsible for taking resource attributes from annotations only. Instead of calling it `automatic_attributes`, it can be something like
```
extract:
  otel_annotations: <bool> // false by default
```

Then, if we want to provide flexibility for annotations extraction, we could add `extract.annotations` with the same syntax as we currently have for `extract.labels`. That is not really needed right now.

Providing such interface would be consistent with the existing one and easier to understand by the end users. Please let me know WDYT.